docs/locale/pot/admin.pot

```diff
@@ -26,2282 +25,1851 @@ msgstr ""
 msgid "Database Entries"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:156
-# 909a16fafb7e417297ad084bf660a6b3
 msgid "We must now change 4 entries in the database. This is easily accomplished with 4 simple SQL commands:"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:173
-# ab52e99bd34e48a8924ac2589b53a8e7
 msgid "Change the site URL"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:201
-# 8d3a3e6504414f6f80d2015c0a6efdbc
 msgid "Only change the first path here!!"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:204
-# 70da393fa49b46ca98d7b60f91c24aa8
 msgid "Check .htaccess"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:206
-# aba20e50fb0c49249cddb7046cd376a0
 msgid "If you have made changes to .htaccess that modify any paths, make sure you update them in the test installation."
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:209
-# 294d68b1d722494abc07895dcfa212f2
 msgid "Update Webserver Config"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:211
-# 2b113757497b4b5790a550cdc1398f2c
 msgid "For this example, you must edit the Apache config to enable a subdomain with a document root of ``/var/www/elgg_test/``. If you plan to install into a subdirectory of your document root, this step is unnecessary."
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:213
-# 29343cf3a5d744029dabb20de0d642cf
 msgid "If you're using Nginx, you need to update server config to match new paths based on ``install/config/nginx.dist``."
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:216
-# ab1979dc29d840e086246b449df95ae6
 msgid "Run upgrade.php"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:218
-# 99493164bc6f4bfaac7005751516c31c
 msgid "To regenerate cached data, make sure to run ``http://test.myelgg.org/upgrade.php``"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:221
-# 457f741fe4a943e3a8b5f2fa5008c3e2
 msgid "Tips"
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:223
-# 76a04f80f11f4c168bdeb9c6118e6684
 msgid "It is a good idea to keep a test server around to experiment with installing new mods and doing development work. If you automate restorations to the ``elgg_test`` database, changing the ``$CONFIG`` values and adding the follow lines to the end of the ``elgg_test/engine/settings.php`` file will allow seamless re-writing of the MySQL database entries."
 msgstr ""
 
 #: ../../admin/duplicate-installation.rst:270
-# ebe4c5e2e06f484c9733c957b5cb2ca3
 msgid ":doc:`backup-restore`"
 msgstr ""
 
 #: ../../admin/getting-help.rst:2
-# 73d5a6f84aae482f8ebf1ba21b0ecabe
 msgid "Getting Help"
 msgstr ""
 
 #: ../../admin/getting-help.rst:4
-# d8baf99e508d4a639ae9fc0b67978654
 msgid "Having a problem with Elgg? The best way to get help is to ask at the `Community Site`_. This site is community supported by a large group of volunteers. Here are a few tips to help you get the help you need."
 msgstr ""
 
 #: ../../admin/getting-help.rst:13
-# 62afaea1a58346d39978e80bc2670ad3
```

Comment:
Nice of transifex to remove these hashes. Was causing a whole lot of useless diff noise...